docs/guides/01-overview.md

```diff
@@ -0,0 +1,173 @@
+          <h1>Add a layer</h1>
+          <button class="github-logo js-source-link"></button>
+        </header>
+        <section>
+          <p class="description open-sans">Add one CARTO layer to your map.</p>
+        </section>
+        <footer class="js-footer"></footer>
+      </div>
+    </aside>
+
+    <script>
+      const map = L.map('map').setView([30, 0], 3);
+      map.scrollWheelZoom.disable();
+
+      L.tileLayer('https://{s}.basemaps.cartocdn.com/rastertiles/voyager_nolabels/{z}/{x}/{y}.png', {
+        maxZoom: 18
+      }).addTo(map);
+
+      const client = new carto.Client({
+        apiKey: 'default_public',
+        username: 'cartojs-test'
+      });
+
+      const source = new carto.source.Dataset(`
+        ne_10m_populated_places_simple
+      `);
+      const style = new carto.style.CartoCSS(`
+        #layer {
+          marker-width: 7;
+          marker-fill: #EE4D5A;
+          marker-line-color: #FFFFFF;
+        }
+      `);
+      const layer = new carto.layer.Layer(source, style);
+
+      client.addLayer(layer);
+      client.getLeafletLayer().addTo(map);
+    </script>
+  </body>
+</html>
+```
+
+[View example]({{site.cartojs_docs}}/examples/#example-add-a-layer).
+
+Even in this simple example, there are a few things to note:
+
+  - We declare the application as HTML5 using the `<!DOCTYPE html>` declaration.
+  - We create a `div` element named "map" to hold the map.
+  - We define a JavaScript function that creates a map in the `div`.
+  - We load the CARTO.js library using a `script` tag.
+
+These steps are explained below.
+
+### Declaring your application as HTML5
+
+We recommend that you declare a true DOCTYPE within your web application. Within the examples here, we've declared our applications as HTML5 using the simple HTML5 DOCTYPE as shown below:
+
+```html
+<!DOCTYPE html>
+```
+
+Most current browsers will render content that is declared with this DOCTYPE in "standards mode" which means that your application should be more cross-browser compliant. The DOCTYPE is also designed to degrade gracefully; browsers that don't understand it will ignore it, and use "quirks mode" to display their content.
+
+We add styles to the map through the file `style.css`, declaring:
+
+```css
+body {
+  margin: 0;
+  padding: 0;
+}
+
+#map {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  z-index: 0;
+}
+```
+
+This CSS declaration indicates that the map container <div> (with id map) should take up 100% of the height of the HTML body.
+
+### Loading the CARTO.js library
+
+To load the Maps JavaScript API, use a script tag like the one in the following example:
+
+```html
+<script src="https://libs.cartocdn.com/carto.js/v4.0.8/carto.min.js"></script>
+```
+
+The URL contained in the script tag is the location of a JavaScript file that loads all of the symbols and definitions you need for using the CARTO.js library. This script tag is required. We are using the minified version of the library.
```

Comment:
@inigomedina do you mean "symbols" like ES6 symbols https://www.keithcirkel.co.uk/metaprogramming-in-es6-symbols/ ? CARTO.js is using those? Just want to make sure this is accurate - I hadn't heard the term before.

Reply:
We don't use Symbols. We can leave it as:

`...that loads all thee code you need for using...`

Reply:
👍